docs/hub/spaces.md

```diff
@@ -28,7 +28,7 @@ Each Spaces environment is limited to 16GB RAM and 8 CPU cores. Organization [su
 
 ## Streamlit and Gradio
 
-Spaces support [Streamlit](https://streamlit.io/) and [Gradio](https://gradio.app/) for quickly building apps in Python. The default environment runs version `"1.0.0"` of Streamlit and the latest version of Gradio. We recommend you try both because they're really awesome! Here are some of our thoughts on Streamlit and Gradio:
+Spaces support [Streamlit](https://streamlit.io/) and [Gradio](https://gradio.app/) for quickly building apps in Python. The default environment runs whichever latest version of Streamlit and Gradio is the current one at the time you create your Space repo. We recommend you try both because they're really awesome! Here are some of our thoughts on Streamlit and Gradio:
```

Comment:
We have it documented [here](https://huggingface.co/docs/hub/spaces#troubleshoot) (only for Streamlit), and then in the Reference below, we list all the supported versions. Small suggestion, otherwise looks good!

```suggestion
Spaces support [Streamlit](https://streamlit.io/) and [Gradio](https://gradio.app/) for quickly building apps in Python. The default environment runs the latest version of Streamlit and Gradio at the time the Space repo is created. We recommend you try both because they're really awesome! Here are some of our thoughts on Streamlit and Gradio:
```